sandy/fy.py

```diff
@@ -234,18 +234,18 @@ def get_mass_chain(self, zam, e):
         --------
         >>> tape_nfpy = sandy.get_endf6_file("jeff_33",'nfpy','all')
         >>> nfpy = Fy.from_endf6(tape_nfpy)
-        >>> nfpy.get_mass_chain(922350, 0.0253).loc[148]
+        >>> nfpy.get_mass_yield(922350, 0.0253).loc[148]
         0.0169029147
         """
         # Filter FY data:
         conditions = {'ZAM': zam, "E": e, 'MT': 454}
         fy_data = self._filters(conditions).data.set_index('ZAP').FY
         chain_data = self._filters({'ZAM': zam, "E": e})
-        S = chain_data.get_mass_chains_sensitivity()
+        S = chain_data.get_mass_yields_sensitivity()
         chain = S.dot(fy_data)
         return chain.rename('mass chain')
 
-    def get_mass_chains_sensitivity(self):
+    def get_mass_yields_sensitivity(self):
```

Comment:
this should be `get_mass_yield_sensitivity`, to be consistent with `get_chain_yield_sensitivity`